### PR TITLE
Fix for "Move the Item header in handbook"

### PR DESCRIPTION
**Pull request by calicomills**
…hnuck26@gmail.com>

Signed-off-by: Jishnu C K <jishnuck@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

Made changes only in CSS in 9 .js files in "/layer5/src/sections/Community/Handbook", each rendering respective handbook item pages. The idea was to move the item heading to the same block as  content of the page. Also reduced the "padding-top" for the content to make the UI look compact.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.
 

image

### CODE REVIEW

**Comment by leecalcote** [comment]
Thank you for providing a screenshot.￼ Some good work here. With respect to reducing the top most padding, please note that this site is whitespace friendly￼. Have a look at this page for an example of the desired amount of margin - https://layer5.io/learn

**Comment by l5io** [comment]
🚀 Preview for commit 64049ba04a539c27660ecfa08f0cf53da6e09144 at: https://6116e5912d7bdb1b894ec9b6--layer5ng.netlify.app

**Comment by calicomills** [comment]
> Thank you for providing a screenshot.￼ Some good work here. With respect to reducing the top most padding, please note that this site is whitespace friendly￼. Have a look at this page for an example of the desired amount of margin - https://layer5.io/learn

okay...I will check and improve on that. What's the process to be followed now..I cancel the PR..make the changes and resubmit?

**Comment by calicomills** [comment]
With the latest changes, I have reverted the changes related to "padding-top", more whitespace is available now.

**Comment by Nikhil-Ladha** [comment]
@leecalcote is there any specific reason to have the `headings` under the community handbook in "black" color? We use the `JUngle Green" for all the headings throughout the site.

**Comment by l5io** [comment]
🚀 Preview for commit 411ea7ca585391b2781687b2134965a4240d0771 at: https://6117b0c84eba5face98e84f8--layer5ng.netlify.app

**Comment by leecalcote** [comment]
@Nikhil-Ladha no, that shouldn't be the case. The handbook should be following along with the rest of the existing format.

**Comment by calicomills** [comment]
@Nikhil-Ladha "secondaryLightColor: "#FAFAFA""...Is this the one?

**Comment by calicomills** [comment]
@Nikhil-Ladha @leecalcote Made the changes to heading colour to follow existing format.

**Comment by Nikhil-Ladha** [comment]
You shouldn't be changing the color and the font-size of the header elements at all( until and unless compulsory) which doesn't seem to be the case here. Please update that in all the places under the community handbook code

**Comment by calicomills** [comment]
> You shouldn't be changing the color and the font-size of the header elements at all( until and unless compulsory) which doesn't seem to be the case here. Please update that in all the places under the community handbook code

The header wasn't present earlier, it's a newly added element. What is the prescribed colour and size for the header in the existing format?

**Comment by Nikhil-Ladha** [comment]
If it's newly added then don't define new size and color for it.
The generic styles can ve found in this [file](https://github.com/layer5io/layer5/blob/master/src/sections/app.style.js)

**Comment by calicomills** [comment]
> If it's newly added then don't define new size and color for it.
> The generic styles can ve found in this [file](https://github.com/layer5io/layer5/blob/master/src/sections/app.style.js)

Okay thanks, I can get the font size from here, but there's no mention of heading colour in the CSS file. But like you mentioned earlier I could find darkJungleGreen used for headings in other pages.

**Comment by Nikhil-Ladha** [comment]
The color is defined on line: 56

File: src/sections/Community/Handbook/about.js [hunk 1]
```diff
@@ -10,6 +10,12 @@ import TOC from "../../../components/handbook-navigation/index";
 const IntroWrapper = styled.div`
     padding: 3rem 20rem;
     margin-top: -25rem; 
+    h1{
+      color: ${props => props.theme.darkJungleGreenColor};
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/about.js:14]: Why define the same CSS again? That's not needed.

Comment (Nikhil-Ladha) [reply src/sections/Community/Handbook/about.js:14]: Applicable for font-size and font-weight as well

Comment (calicomills) [reply src/sections/Community/Handbook/about.js:14]: alright..Got it! ...So only thing I need to keep is the margin-bottom property.

**Comment by l5io** [comment]
🚀 Preview for commit a00cf4510fc0813de6ab591bdcb8cb2d5e5a6f24 at: https://611bfd3fe4a29f070c671f22--layer5ng.netlify.app

**Review by Anita-ihuman — APPROVED** [review]
LGTM

**Comment by warunicorn19** [comment]
@calicomills Please revert the changes to json file.

**Comment by calicomills** [comment]
> @calicomills Please revert the changes to json file.

I have only made changes to .js files, Isn't the json an autogenerated file?

**Comment by Nikhil-Ladha** [comment]
> > @calicomills Please revert the changes to json file.
> 
> I have only made changes to .js files, Isn't the json an autogenerated file?

Yeah, that can be ignored @warunicorn19 .

**Comment by Nikhil-Ladha** [comment]
It's just that, the preview doesn't look like the screenshot shared above 🤔

**Comment by Nikhil-Ladha** [comment]
Also, @Anita-ihuman, please make sure that we keep the consistency in colors for the handbook, starting with the removal of the `black` color for headings.

**Comment by calicomills** [comment]
> It's just that, the preview doesn't look like the screenshot shared above 🤔

I did add more whitespace above the headings after that.

**Comment by calicomills** [comment]
@Nikhil-Ladha @leecalcote @warunicorn19 Please review.

File: src/sections/Community/Handbook/about.js [hunk 2]
```diff
@@ -8,22 +8,72 @@ import Bullet from "./images/active.png";
...
 
 
+const IntroWrapper = styled.div`
+    padding: 3rem 20rem;
+    margin-top: -25rem; 
+    h1{
+      margin-bottom: 2rem;
+    }
+    h2{
+      color:black;
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/about.js:18]: Why these colors are being added again?

Comment (warunicorn19) [reply src/sections/Community/Handbook/about.js:18]: @Nikhil-Ladha i think what he's working on has already been fixed, i think. And also the wrapper he's working on has been deleted already

**Pull request closed without merging**